config/clusters/utoronto/default-staging.values.yaml

```diff
@@ -4,12 +4,6 @@ jupyterhub:
     tls:
       - hosts: [staging.utoronto.2i2c.cloud]
         secretName: https-auto-tls
-  custom:
-    homepage:
-      gitRepoBranch: "utoronto-staging"
-  singleuser:
-    image:
-      tag: "14320bae73a0"
```

Comment:
Also in the other PR, but probably fine?

Reply:
@consideRatio ah correct. But I've removed it from here anyway just for cleanliness.